app/src/store/voice/call/activeCall.ts

```diff
@@ -47,8 +47,11 @@ export const disconnectActiveCall = createAsyncThunk<
     if (!call) {
       return rejectWithValue('CALL_UUID_NOT_FOUND');
     }
-
-    await call.disconnect();
+    try {
```

Comment:
Since we're catching these now, these thunks will never really "fail". So if you try matching fulfillment or rejection using the Redux toolkit matchers, they won't give us what we want.

Logging the error here doesn't also provide too much, the default behavior of an unhandled Promise reject is effectively the same as what we're doing here - arguably more useful. I think we need to use `rejectWithValue` here and have more information in the store about the failure reason. See the above recommendation with having a `reason` string.